src/ngSanitize/sanitize.js

```diff
@@ -158,7 +158,7 @@ function $SanitizeProvider() {
     return function(html) {
       var buf = [];
       htmlParser(html, htmlSanitizeWriter(buf, function(uri, isImage) {
```

Comment:
thanks. Waiting on a last review from @IgorMinar  and @gkalpak before merging. Did you @rjamet test this on Google3 already?